src/pages/events/index.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
 import EventsContainer from './styled'
 import NavBar from '../../components/navbar'
-import Parceires from '../../components/parceires'
+import Partners from '../../components/partners'
 import Footer from '../../components/footer'
-import Events from '../../components/events-carrosel'
+import CarouselEvents from '../../components/carousel-events'
 
 export const EventsPage = () => {
   return (
     <EventsContainer>
       <NavBar />
-      <Events />
-      <Parceires />
+      <CarouselEvents />
```

Comment:
Fiquem totalmente a vontade para ignorar, mas acho que esse nome seria mais natural na seguinte forma (estou sendo chato não me matem pfv):

```suggestion
      <EventsCarousel />
```